other/bootstrap_daemon/tox-bootstrapd.service

```diff
@@ -4,11 +4,8 @@ After=network.target
 
 [Service]
 Type=forking
-PermissionsStartOnly=true
-ExecStartPre=-/bin/mkdir /var/run/tox-bootstrapd -p
-ExecStartPre=/bin/chown tox-bootstrapd:tox-bootstrapd -R /var/run/tox-bootstrapd
 WorkingDirectory=/var/lib/tox-bootstrapd
-ExecStart=/usr/local/bin/tox-bootstrapd /etc/tox-bootstrapd.conf
+ExecStart=/usr/bin/tox-bootstrapd /etc/tox-bootstrapd.conf
```

Comment:
>How will I get this reverted? I just make another commit with the path fixed?

Just make a commit and rebase (squash) it.

>Anyway, the systemd files get copied manually. Maybe I better make some amendments in the readme at other/bootstrap_daemon/README.md

Which amendments? any software you build from the source is set to install in `/usr/local/bin`, it's not Tox-specific, it's universal, there is no need to mention that.
If it's not built from source by the user, i.e. the user gets tox-boostrapd from distribution's package repository, then it's package maintainer's job to make sure that the software plays by distribution's rules and also that it installs in `/usr/bin`.